### DISCUSSION TITLE
Unmaintained extension

### DISCUSSION THREAD

**Discussion by ajhawkings** [microsoft/vscode-discussions#206] [Extension Development QnA] [answered]
Does anyone know if there any way to get in contact with the VS code team about an [unmaintained extension pack](https://marketplace.visualstudio.com/items?itemName=waderyan.nodejs-extension-pack)? It was created by a former Microsoft employee and has 700k downloads, but has not been updated in 6 years and the author is no longer merging pull requests e.g. to remove a depreciated extension (https://github.com/waderyan/nodejs-extension-pack/pull/8). Is there anything that can be done from vscode's side to contact the author or update the extension? Thanks.

**Comment by alefragnani** [comment]
Hi @ajhawkings ,

I guess the best location to ask for this is in the VS Marketplace repo (https://github.com/microsoft/vsmarketplace). But, based on [this similar question made in January](https://github.com/microsoft/vsmarketplace/issues/285#issuecomment-1003900106) the team didn't have a policy for this yet. 

Maybe something has changed since then, so I suggest you to ask this question there.

Hope this helps

**Comment by isidorn** [comment, marked as answer]
The best would be to comment here https://github.com/microsoft/vscode-discussions/discussions/1
And to make sure to file an issue in the repo of the extension pack so we get confirmation from extension author that it is indeed deprecated.

**Reply by ajhawkings** [reply]
Hi @isidorn, the extension pack has 700k users who would have to search for an alternative if it was depreciated - is there any way to just change the existing one? I don't think opening an issue in the repo would work as well, as the developer of the extension pack hasn't responded to any PRs or issues for 5+ years. Thanks!
